assembl/scripts/ini_files.py

```diff
@@ -78,9 +93,42 @@ def ensureSection(config, section):
         config.add_section(section)
 
 
+def fill_template(template, config, output=None):
+    if not exists(template):
+        template = join(local_code_root, 'templates', 'system', template)
+    if not exists(template):
+        raise RuntimeError("Missing template")
+    config['here'] = config.get('here', os.getcwd())
+    if template.endswith('.tmpl'):
+        with open(template) as tmpl:
+            result = tmpl.read() % config
+    elif template.endswith('.jinja2'):
+        from jinja2 import Environment
+        env = Environment()
+        with open(template) as tmpl:
+            tmpl = env.from_string(tmpl.read())
+        # Boolean overloading
```

Comment:
Because you bypassed the Jinja2 read? Why?

Reply:
Because jinja does not translate the 'false' string into the False value. It's actually a jinja bug, mentioned here: https://github.com/ansible/ansible/issues/14983

Reply:
I would comment this link here, so that it can be checked, and this code be altered if/when another developer reviews it.